app/assets/stylesheets/application.scss

```diff
@@ -1,5 +1,7 @@
 @import url(//fonts.googleapis.com/css?family=Gloria+Hallelujah);
 
+@import "components/nav";
 body {
   font-family: 'Gloria Hallelujah', Helvetica, Arial;
```

Comment:
@noveng05 and hunter's PR is gonna be unhappy with this! will either have merge conflicts or a missing set of fonts

https://github.com/rubyandrose/grenache/pull/2/files